### PR TITLE
chore: Dependencies & Docs

### PR DESCRIPTION
**Pull request by tehhowch**
The "merge-me" action is failing until @christian-bromann makes some changes, so the dev deps are out of date.

 - update lockfile version to v2
 - update dev deps to latest v7 WDIO
 - fix reported vulnerabilities in dev deps
 - update changelog file
 - update nvmrc file to the current v14 LTS version.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-12.16.3
+14.21.1
```
Comment (christian-bromann) [inline .nvmrc:1]: Why not going to at least `v16`? I personally would do even `v18`

**Comment by christian-bromann** [comment]
> The "merge-me" action is failing until @christian-bromann makes some changes,

What changes are you expecting me do to? I don't have a replacement for it yet. Feel free to use anything that works.

File: .nvmrc (hunk 1 above)
Comment (tehhowch) [reply .nvmrc:1]: This repo historically supports very legacy tooling, so I went with the oldest non-eol LTS. (Personally, I use `n`, so the setting doesn't affect me anyway. I included it for completeness.)

**Comment by tehhowch** [comment]
> > The "merge-me" action is failing until @christian-bromann makes some changes,
> 
> What changes are you expecting me do to? I don't have a replacement for it yet. Feel free to use anything that works.

When I get time over the next week I plan to introduce an alternative that uses only GitHub's actions. I had tagged you since you had added it.

**Comment by christian-bromann** [comment]
Cheers! Yeah would love if we could roll out a better action to all projects as it takes me almost an hour every week to keep up with all these manual merges.

**Comment by christian-bromann** [comment]
Please go ahead and merge.

**Comment by tehhowch** [comment]
> Please go ahead and merge.
image

I'm not a maintainer/collaborator here, which is why I end up pinging you and @chmanie to merge things

**Comment by christian-bromann** [comment]
Oh sorry, I just invited you. Welcome to the team!

**Comment by christian-bromann** [comment]
If `main` branch is green feel free to trigger the release workflow with the appropriate semver bump. Let me know if you have any questions.

**Comment by chmanie** [comment]
> Oh sorry, I just invited you. Welcome to the team!

Happy this finally happened :) welcome @tehhowch and thanks for all the great work!

**Pull request merged**